python/openmldb_tool/diagnostic_tool/diagnose.py

```diff
@@ -79,6 +84,38 @@ def status(args):
         ), f"not all components in conf file are online, check the previous output"
         print(f"all components in conf file are online")
 
+    if args.conn:
```

Comment:
use the same name `conn`, you can rename the Connector

Reply:
renamed to connect